tests/rptest/tests/cluster_features_test.py

```diff
@@ -172,20 +172,7 @@ def test_license_upload_and_query(self):
         resp = self.admin.get_license()
         assert resp['loaded'] is True
         assert resp['license'] is not None
-
-        def is_equal_to_license_properties(license_contents,
-                                           license_properties):
-            """Compares the values within first parameters map to a response
-            from the redpanda admin server"""
-            days_left = (license_contents['expires'] -
-                         datetime.date.today()).days
-            return license_properties['format_version'] == license_contents['format_version'] and \
-                license_properties['org'] == license_contents['org'] and \
-                license_properties['type'] == license_contents['type'] and \
-                license_properties['expires'] == days_left
-
-        assert is_equal_to_license_properties(license_contents,
-                                              resp['license']) is True
+        assert expected_license_contents == resp['license'], resp['license']
```

Comment:
@graphcareful what is this assertion testing? it looks like it is comparing a dictionary to a tuple

Reply:
No that syntax just prints the value of `resp[license]` in the event the assertion triggers